bioimageio/core/predict.py

```diff
@@ -0,0 +1,32 @@
+import argparse
+import sys
+
+import numpy as np
+import torch.cuda
+import xarray as xr
+
+from bioimageio.core.prediction_pipeline import create_prediction_pipeline
+from bioimageio.spec import import_package
+
+parser = argparse.ArgumentParser()
+parser.add_argument("-m", "--model", help="bioimage model package (zip file)", required=True)
+parser.add_argument("image", nargs="?", help="image to process (.npy)")
+parser.add_argument("-o", "--output", nargs="?", help="output image (.npy)", required=True)
+
+
+def main():
+    args = parser.parse_args()
+    bioimageio_model = import_package(args.model)
+    prediction_pipeline = create_prediction_pipeline(
+        bioimageio_model=bioimageio_model, devices=["cuda" if torch.cuda.is_available() else "cpu"]
+    )
+
+    input_tensor = np.load(args.image)
+    tagged_data = xr.DataArray(input_tensor, dims=tuple(bioimageio_model.input_axes))
+    res = prediction_pipeline.forward(tagged_data)
+    np.save(args.output, res.data)
```

Comment:
It would be nice to derive the type of the output data from the file extension, e.g. use `imageio` if the ending is `tif` or `png`. But we can also do this in a later PR.

Reply:
I suggest we keep this separate

Reply:
yes, I agree